api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -536,11 +536,10 @@
     Attribute<String> LIBRARY_JARS = new Attribute<>(String2String.getInstance());
 
     /**
-     * This configuration file can be used to over-ride the default Yarn configuration.
-     * For example, this can be used to provide custom SSL parameters in the configuration.
-     * Note that this file needs to be present on the node.
+     * SSL Configuration string property. Currently used to specify SSL parameters for
```

Comment:
For the second part of the statement. How about just saying Used to specify SSL configuration for stram web services.